### ISSUE TITLE
Support for Amber

### ISSUE THREAD

**Issue by greenart7c3** [dluvian/Nozzle#62]
Do you have any plans to add support for amber?

https://github.com/greenart7c3/amber

**Comment by dluvian** [comment]
I'd like to add a third party signer but don't want to be restricted to just Amber. Is it possible to add Amber support without explicitly including `com.greenart7c3.nostrsigner`?

**Comment by greenart7c3** [comment]
Yes, in amethyst we do that when you call the get public key the signer return its package name so you can save it
